public/employers/stanford-health-care.yml

```diff
@@ -31,10 +31,13 @@ citations:
     sources:
       - source: Stanford Health Care
         link: https://stanfordhealthcare.org/stanford-health-care-now/2020/novel-coronavirus/cdc-advice-on-personal-protective-equipment-ppe.html # Unknown date.
-  - summary: Stanford Health Care sent a letter to employees to notify them that they must either take a 20% pay cut or use paid time off for the period of April 27 through July 4.
+  - summary: Stanford Health Care sent a letter to employees to notify them that they must either take a 20% pay cut or use paid time off for the period of April 27 through July 4. In a response, employees noted that executives have not taken any pay cuts to offset the losses.
```

Comment:
I feel like this should be a -2 instead. Most people will find it hard to survive more than two months without a regular income.

Reply:
So typically I use -1 for furloughs, layoffs, or pay cuts less than about 30%. This basically indicates poor planning on the part of the company, and sometimes an unwillingness to cut executive pay (though somethigns it balances out with a +1 for "CEO and executives took pay cut").

I usually reserve -2 for actively endangering people's safety, covering things up, or breaking the law.

In this particular case, they can continue working and being paid at only 20%, or they can take **paid** time off. Seems more like a -1 to me. Thoughts?

Reply:
That's fair. The other reason I think this is a -2 is since you have 2 items in one summary in my mind, but I can see how you can treat it as one thing. We can keep it as -1.

Reply:
Actually since a lot of the other claims have CEO/Execs pay separated with employee pay, I think we should be consistent here as well.